script.js

```diff
@@ -1,4 +1,67 @@
+// User inputs one of "scissors", "paper", or "stone",
+// Program internally randomly chooses scissors, paper, or stone,
+// Program outputs whether the user won, the program won, or it's a draw.
+// Rules: scissors beats paper, paper beats stone, and stone beats scissors.
+// If both parties choose the same object, it's a draw.
+
+// reverse scissors paper stone game
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var computer = generateScissorsPaperStone();
+  console.log("Computer is choosing");
+  console.log(computer);
```

Comment:
for lines 10-11, if your aim is to show what computer is choosing with line 10 as a label, you can put it as : console.log("Computer choose", computer)